src/Sylius/Bundle/ShopBundle/Resources/views/error403.html.twig

```diff
@@ -1,10 +1,2 @@
-{% extends '@SyliusShop/layout.html.twig' %}
-
-{% block content %}
-    <h2 class="ui center aligned icon header">
-        <img src="{{ asset('assets/shop/img/logo.png') }}" alt="Sylius logo" class="ui small image" />
-        <br/><br/>
-        {{ 'sylius.ui.the_page_you_are_looking_for_is_forbidden'|trans }}
-    </h2>
-
-{% endblock %}
+{% set message = 'sylius.ui.the_page_you_are_looking_for_is_forbidden'|trans %}
+{% include '@SyliusUi/error.html.twig' with {'status_code': '403', 'status_text': message}%}
```

Comment:
Can you revert these changes? Previously created errors should stay as they were